doc/manual/nuc_handbook.rst

```diff
@@ -0,0 +1,257 @@
+
+    programs.openssh.enable = true;
+
+both installs the ``openssh`` package and configures and starts the ``sshd`` service.
+
+Nix and NixOS tips
+------------------
+
+Installing additional packages (ad-hoc)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Often you might find yourself wanting access to a tool or a piece of software for a single use case or a limited time, without really wanting to install it into your system permanently. Nix provides a very convenient solution for this. Any package in the `Nixpkgs repository <https://search.nixos.org/packages>`_ can be installed temporarily, into a particular shell, with the command: ::
+
+    $ nix-shell -p <pkg-name>
+
+.. warning::
+
+    Somewhat unfortunately, the two commands ``nix-shell`` and ``nix shell`` both exist in NixOS and **are not synonymous**. Note in this case the use of the hyphen. To start a shell from a *flake*, use ``nix shell`` with no hyphen. A similar distinction applies for ``nix-build`` and ``nix build``.
+
+Much as with flakes, this may take some time at a first use for large packages, but if you end up calling it repeatedly, subsequent invocations will use the Nix cache and run almost instantly. Also much as with flakes, the package is not 'installed' in a permanent sense and will disappear once the shell is closed. This is also useful in order to be able to test packages quickly before installing them into the environment.
+
+Flakes and builds
+^^^^^^^^^^^^^^^^^
+
+Running NixOS is perfectly compatible with the other Nix features used by ARTIQ. In particular, using ``nix shell`` or ``nix develop`` with the various ARTIQ flakes, as described in :doc:`installing` and :doc:`flashing`, can continue to be very useful, especially to access different versions of ARTIQ, even simultaneously. It is also perfectly possible to install packages into your Nix profile. All of these are ways to define and access certain environments, with access to certain sets of packages, none of which will overwrite each other.
+
+Note that, for instance, if you have installed ARTIQ ``release-8`` into your environment, but run ``nix shell`` on the ARTIQ ``release-7`` flake, that *specific shell* will run ``release-7`` commands, whereas all others will continue to use the system default environment, with ``release-8``. You can check on this kind of behavior yourself with ``--version``.
+
+.. _nix-collect-garbage :
+
+Garbage collection
+^^^^^^^^^^^^^^^^^^
+
+As also noted in :doc:`installing`, Nix caches generously in ``/nix/store``, which should ensure that packages only ever have to be fetched and built once when reused repeatedly or between different environments. Old NixOS generations are also stored, and will remain available in the boot menu as long as they are stored. With time however this cache can occupy large amounts of storage space. To clear the cache, run: ::
```

Comment:
```suggestion
As also noted in :doc:`installing`, Nix stores everything in ``/nix/store``, the so-called ``Nix Store``. This ensures packages only ever have to be fetched and built once, even when reused repeatedly between different environments. Old NixOS generations are also stored in there, and will remain available in the boot menu upon future boots. With time however this can occupy large amounts of storage space. To garbage-collect (clean up files not used by NixOS) the Nix Store, run: ::
```

Calling this caching is misleading.